compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -225,7 +225,8 @@ impl<'hir> Map<'hir> {
         self.tcx.definitions_untracked().iter_local_def_id()
     }
 
-    pub fn opt_def_kind(self, local_def_id: LocalDefId) -> Option<DefKind> {
+    /// Do not call this function directly.  The query should be called.
```

Comment:
minor nit: remove double space
```suggestion
    /// Do not call this function directly. The query should be called.
```